app/assets/javascripts/discourse/tests/acceptance/composer-test.js

```diff
@@ -217,15 +217,7 @@ acceptance("Composer", function (needs) {
     textarea.selectionStart = textarea.value.length;
     textarea.selectionEnd = textarea.value.length;
 
-    // Testing keyboard events is tough!
-    const event = document.createEvent("Event");
-    event.initEvent("keydown", true, true);
-    event[`${PLATFORM_KEY_MODIFIER}Key`] = true;
-    event.key = "B";
-    event.keyCode = 66;
-
-    textarea.dispatchEvent(event);
-    await settled();
+    await triggerKeyEvent(textarea, "keydown", "B", metaModifier);
```

Comment:
Love it! "testing keyboard events is tough + 7 lines of hack" --> "single line helper" 💪